api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@xiphiaz I'm not sure I completely follow along now? Haha. But I think we have it implemented correctly now, or do I need to adjust anything?